src/DeVinci_frontend/components/SelectModelOption.svelte

```diff
@@ -2,7 +2,7 @@
   import * as webllm from "@mlc-ai/web-llm";
   import { onMount } from "svelte";
   import { location, push } from 'svelte-spa-router';
-
+  import { addDownloadedModel } from '../helpers/modelStore';
```

Comment:
this functionality is implemented in localStorage.js setLocalFlag already, best to use the existing function for this.
e.g. in SelectModelOption.svelte
// Set flag that this model has been downloaded
      const flagObject = {
        modelId: modelOptionId,
      };
      setLocalFlag("downloadedAiModels", flagObject);